js/ui/viewSelector.js

```diff
@@ -200,13 +197,6 @@ const ViewSelector = new Lang.Class({
                 }
             }));
 
-        Main.wm.addKeybinding('toggle-application-view',
-                              new Gio.Settings({ schema_id: SHELL_KEYBINDINGS_SCHEMA }),
-                              Meta.KeyBindingFlags.NONE,
-                              Shell.ActionMode.NORMAL |
-                              Shell.ActionMode.OVERVIEW,
-                              Lang.bind(this, this._toggleAppsPage));
-
         Main.wm.addKeybinding('toggle-overview',
                               new Gio.Settings({ schema_id: SHELL_KEYBINDINGS_SCHEMA }),
                               Meta.KeyBindingFlags.NONE,
```

Comment:
This should call toggleWindows() too, I'll change both things as one fixup commit